model/database/cosmosdb-metrics.yaml

```
@@ -26,3 +26,68 @@ groups:
       - ref: server.port
         requirement_level:
           conditionally_required: If using a port other than the default port for this DBMS and if `server.address` is set.
+
+  - id: metric.db.client.cosmosdb.request.duration
+    type: metric
+    metric_name: db.client.cosmosdb.request.duration
+    brief: "End to end Latency of Azure Cosmos DB Backend Service network call"
```

I assume we measure the latency on the client, but the brief gives an impression it's measured on the server

```suggestion
    brief: "End to end latency of a network call made to Azure Cosmos DB service"
```